kimhaejung/app.js

```diff
@@ -0,0 +1,57 @@
+require("dotenv").config();
+
+const express = require("express");
+const cors = require("cors");
+const logger = require("morgan");
+
+const { DataSource } = require("typeorm");
+
+const port = process.env.PORT || 3000;
```

Comment:
해당 변수를 선언하는 부분이 상단부에 위치해도 좋지만,
실제 `app.listen()` 함수에서만 사용되기 때문에 해당 코드 라인 바로 윗줄로 이동하여 작성하는 것도 고려해보면 좋을 것 같아요!

```suggestion
const port = process.env.PORT || 3000;
app.listen(port, () => { ... })
```